playwright/test/exhibition-guides.test.ts

```diff
@@ -38,33 +38,30 @@ test('(2) | Does not redirect if user does not come from a QR code', async ({
   page,
 }) => {
   // Jason with the QR code params and Stop #2
-  await digitalGuide(
-    `${newJasonGuideId}?stopNumber=2`,
-    context,
-    page,
-    bslCookie
-  );
+  await digitalGuide(`${jasonGuideId}?stopNumber=2`, context, page, bslCookie);
 
   // Nothing happens, URL is the same
   await expect(page).toHaveURL(
-    /\/guides\/exhibitions\/ZthrZRIAACQALvCC[?]stopNumber=2/
+    /\/guides\/exhibitions\/jason-and-the-adventure-of-254[?]stopNumber=2/
   );
 });
 
-test('(3) | If first stop, redirects to preferred type landing page instead of [type]/1', async ({
+test('(3) | If first stop, redirects to preferred type landing page instead of [type]/jasonGuideId', async ({
   context,
   page,
 }) => {
-  // Jason with the QR code params and Stop #1
+  // Jason with the QR code params and Stop #jasonGuideId
```

Comment:
and here? I'll stop but might be worth reviewing the file. Sorry it was for me to change 😅

Reply:
ha, looks like I somehow managed to do a find and replace against '1'